lib/run_loop/app.rb

```diff
@@ -71,6 +86,16 @@ def self.valid?(app_bundle_path)
       true
     end
 
+    # @!visibility private
+    def self.cached_app_on_simulator?(app_bundle_path)
```

Comment:
There seem to be some insight hidden in the reason-d'être for this method. Can you add a comment about why the check is needed and exactly how the check works?

Reply:
Done.